cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/InstructionTag.qll

```diff
@@ -186,6 +187,8 @@ string getInstructionTagId(TInstructionTag tag) {
   or
   tag = BoolConversionCompareTag() and result = "BoolConvComp"
   or
+  tag = ResultCopyTag() and result = "ResultCopy"
+  or
```

Comment:
This was a drive-by fix I did while debugging. This tag didn't have a result previously.